apps/prairielearn/src/pages/instructorAssessmentInstances/instructorAssessmentInstances.html.ts

```diff
@@ -0,0 +1,460 @@
+      </p>
+      <ul>
+        <li>08:00 - student starts assessment;</li>
+        <li>08:30 - student submits answer;</li>
+        <li>09:00 - student submits answer;</li>
+        <li>(gap of more than one hour)</li>
+        <li>11:00 - student submits answer;</li>
+        <li>11:30 - student submits answer;</li>
+        <li>12:00 - student submits answer.</li>
+      </ul>
+      <p>
+        In the above example, the "duration" would be 2 hours: one hour from 08:00 to 09:00, and
+        another hour from 11:00 to 12:00. The two-hour gap between 09:00 to 11:00 is not counted as
+        part of the duration.
+      </p>
+      <p>
+        <strong>For Exam assessments</strong>, a student is considered to be actively working
+        between the start of the assessment and the last submission, regardless of any potential
+        inactivity. For the same example above, the "duration" would be 4 hours, from 08:00 to
+        12:00. The two-hour gap is not considered inactivity, since it is assumed that this kind of
+        assessment requires students to be active for the duration of the assessment.
+      </p>
+    `,
+    footer: html`
+      <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+    `,
+  });
+}
+
+function TimeRemainingHelpModal() {
+  return Modal({
+    id: 'time-remaining-help',
+    title: 'Time Remaining',
+    body: html`
+      <div class="modal-body">
+        <p>
+          For open assessments with a time limit, this column will indicate the number of minutes
+          (rounded down) the student has left to complete the assessment. If the value is
+          <strong>&lt; 1 min</strong>, the student has less than one minute to complete it. This
+          column may also contain one of the following special values.
+        </p>
+        <ul>
+          <li>
+            <strong>Expired</strong> indicates the assessment time limit has expired, and will be
+            automatically closed as soon as possible. If an assessment is Expired for a prolonged
+            period of time, this typically means the student has closed their browser or lost
+            connectivity, and the assessment will be closed as soon as the student opens the
+            assessment. No further submissions are accepted at this point.
+          </li>
+          <li>
+            <strong>Closed</strong> indicates the assessment has been closed, and no further
+            submissions are accepted.
+          </li>
+          <li>
+            <strong>Open (no time limit)</strong> indicates that the assessment is still open and
+            accepting submissions, and there is no time limit to submit the assessment (other than
+            those indicated by access rules).
+          </li>
+        </ul>
+      </div>
+    `,
+    footer: html`
+      <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+    `,
+  });
+}
+
+function deleteAssessmentInstanceModal({
```

Comment:
As always, let's use `PascalCase` (this applies to below functions as well).

Reply:
Can't believe I still did this. Updated here and below to `PascalCase` in [0b3aa97](https://github.com/PrairieLearn/PrairieLearn/pull/9725/commits/0b3aa97bfea69bfe855c4af0ac526d99182d6f6b).